tokio/src/macros/support.rs

```diff
@@ -1,6 +1,11 @@
 cfg_macros! {
+    #[doc(inline)]
     pub use crate::future::poll_fn;
+
+    #[doc(inline)]
     pub use crate::future::maybe_done::maybe_done;
+
+    #[doc(inline)]
     pub use crate::util::thread_rng_n;
 }
```

Comment:
This module is marked `#[doc(hidden)]` in the `lib.rs` file.

Reply:
Didn't notice that. I have updated the pull request to remove those unnecessary `#[doc(inline)]`.